modules/virt-installing-virtctl-client-yum.adoc

```diff
@@ -1,36 +1,27 @@
 // Module included in the following assemblies:
 //
-// * virt/install/virt-installing-virtctl.adoc
+// * virt/virt-using-the-cli-tools.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-installing-virtctl-client-yum_{context}"]
-= Installing virtctl on {op-system-base}
+= Installing the virtctl RPM on {op-system-base} 8
 
-You install the `virtctl` RPM on {op-system-base-full} by enabling the {VirtProductName} repository and installing the `kubevirt-virtctl` package.
+You install the `virtctl` RPM on {op-system-base-full} 8 by enabling the {VirtProductName} repository and installing the `kubevirt-virtctl` package.
 
 .Prerequisites
 
 * Each host in your cluster must be registered with Red Hat Subscription Manager (RHSM) and have an active {product-title} subscription.
 
 .Procedure
 
-* Enable the appropriate {VirtProductName} repository for your operating system by using the `subscription-manager` CLI tool:
-
-** {op-system-base} 9:
-+
-[source,terminal]
-----
-# subscription-manager repos --enable cnv-4.13-for-rhel-9-x86_64-rpms
-----
-
-** {op-system-base} 8:
+. Enable the {VirtProductName} repository for your operating system by using the `subscription-manager` CLI tool:
 +
 [source,terminal]
 ----
 # subscription-manager repos --enable cnv-4.13-for-rhel-8-x86_64-rpms
 ----
 
-* Install the `kubevirt-virtctl` package:
+. Install the `kubevirt-virtctl` package:
```

Comment:
```suggestion
. Install the `kubevirt-virtctl` package by running the following command:
```
Steps that introduce a command need an explicit instruction to "run" or "enter" the command. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command